lib/bosh_deployment_resource/bosh.rb

```diff
@@ -10,6 +10,15 @@ def initialize(target, ca_cert, auth, command_runner=CommandRunner.new)
       @ca_cert = ca_cert
       @auth = auth
       @command_runner = command_runner
+      @no_redact = false
+    end
+
+    def enable_redact()
```

Comment:
I don't think this should be stateful. If instead it's an argument to `deploy` these aren't necessary.